op-supervisor/supervisor/backend/syncnode/node.go

```diff
@@ -342,6 +347,32 @@ func (m *ManagedNode) resetSignal(errSignal error, l1Ref eth.BlockRef) {
 	}
 }
 
+func (m *ManagedNode) sendReset() {
+	ctx, cancel := context.WithTimeout(m.ctx, internalTimeout)
+	defer cancel()
+
+	u, err := m.backend.LocalUnsafe(ctx, m.chainID)
+	if err != nil {
+		m.log.Warn("Failed to retrieve local-unsafe", "err", err)
+		return
+	}
+	s, err := m.backend.CrossSafe(ctx, m.chainID)
+	if err != nil {
+		m.log.Warn("Failed to retrieve cross-safe", "err", err)
+		return
+	}
+	f, err := m.backend.Finalized(ctx, m.chainID)
+	if err != nil {
+		m.log.Warn("Failed to retrieve finalized", "err", err)
+		return
+	}
```

Comment:
It's possible, especially in test scenarios, for Finalized to not yet be initialized. When a Reset occurs here, the Finalized may return an error describing "can't find finalized, don't know what the first finalized L1 block is" or something.

We should catch that and prevent it by just using a 0-height for finalization if that occurs, since it is possible to want to reset when the services are too fresh for finality.